style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.hero {
+  background-image: url(assets/background-shapes.png);
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
@hrusamaali I would be better if you check you img url in order to make your website 's background look like to figma 👍